doc/source/images/generate_diagram.py

```diff
@@ -173,7 +174,7 @@ def generate_diagram(filename, rect_pos, rect_width, figsize):
             else:
                 t1 = attrname
 
-            if attrtype == pq.Quantity:
+            if attrtype == un.Quantity:
```

Comment:
Use the `units` module only for importing units, not for the `quantities.Quantity` class. This should still be `pq.Quantity`